docs/notebooks/tutorials/bootstrap_current.ipynb

```diff
@@ -0,0 +1,958 @@
+{
```

Comment:
<p>I think you have temperature and density reversed here</p><p>Also units for temperature should be eV</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>